specification/workloads/resource-manager/Microsoft.Workloads/preview/2021-12-01-preview/phpWorkloads.json

```diff
@@ -0,0 +1,1362 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-12-01-preview",
+    "title": "Workloads Client",
+    "description": "Workloads client provides access to various workload operations"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Workloads/phpWorkloads": {
```

Comment:
No CRUD on subscription level phpWorkloads?

Reply:
that would be below API

"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Workloads/phpWorkloads/{phpWorkloadName}"


Note: <b>Please follow the link to find more details on the swagger existing in [azure-rest-api-specs-pr: private repo swagger](https://github.com/Azure/azure-rest-api-specs-pr/tree/RPSaaSMaster/specification/workloads/resource-manager/Microsoft.Workloads). That has already been reviewed by respective teams and is in private preview right now.<b>